.github/workflows/ci-unix-static.yml

```diff
@@ -68,7 +73,7 @@ jobs:
         path: ext, build/_deps
         key: ${{ runner.os }}-${{ matrix.build-type }}-unix-static-${{ hashFiles('cmake/Modules/*', 'ext/*.cmd', 'ext/*.sh') }}
     - name: Setup cmake
-      if: matrix.os == 'ubuntu-latest'
+      if: matrix.os == 'ubuntu-24.04'
```

Comment:
Nit: I think we can test `runner.os == 'Linux'` instead.

Reply:
Done